website/docs/r/vdc.html.markdown

```diff
@@ -0,0 +1,124 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vdc"
+sidebar_current: "docs-vcd-resource-vdc"
+description: |-
+  Provides a vCloud Director Provider VDC resource. This can be used to create and delete an VDC.
```

Comment:
1. "Provider VDC"? Should be "Organization VDC".
2. "an VDC" -> "a VDC"

Reply:
fixed